.github/renovate.json5

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
We have the new file here (probably need to rebase)
https://github.com/camunda/camunda-self-managed/blob/63ec09d0b9bd804e34d36503fe10f0a6e2319efc/.github/renovate.json5#L7

Reply:
I rebased